test/report.js

```diff
@@ -48,8 +48,6 @@ describe('Concur Report Tests', function() {
     });
   });
 
-  var reportId;
```

Comment:
Actually, I removed this line because the linter caught it as a duplicate declaration with line 11.